notesapi/v1/tests/test_views.py

```diff
@@ -959,6 +959,26 @@ def test_pagination_page_start(self, page, start, previous_page, next_page):
             start=start
         )
 
+    @ddt.unpack
+    @ddt.data(
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد عمار", "tags": [u"عمار", u"Muhammad", u"محمد"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"محمد", "tags": [u"محمد", u"Muhammad"]},
+        {"text": u"Ammar محمد عمار Muhammad", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"Muhammad Ammar", "search": u"عمار", "tags": [ u"ammar", u"عمار"]},
+        {"text": u"محمد عمار", "search": u"Muhammad", "tags": [ u"Muhammad", u"عمار"]}
+    )
```

Comment:
I see.  Thanks @jcdyer, that makes sense.  
@muhammad-ammar I don't think it applies here.  I'd just like to see the cases simplified and clarified.  Maybe you could make the cases more clear by describing the strings in the strings?
```
{"text": u"Text with ascii and υηι¢σ∂є.", "search": u"υηι¢σ∂є", "tags": [u"unmatched"]},
```
I'm not sure, because I'm not clear on the cases you wish to test, which is my original point.